editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -5217,6 +5217,29 @@ void Node3DEditor::_init_indicators() {
 		indicator_mat->set_flag(StandardMaterial3D::FLAG_SRGB_VERTEX_COLOR, true);
 		indicator_mat->set_transparency(StandardMaterial3D::Transparency::TRANSPARENCY_ALPHA_DEPTH_PRE_PASS);
 
+		Ref<Shader> grid_shader = memnew(Shader);
+		grid_shader->set_code("\n"
```

Comment:
Sending the first `"\n"` to the next line should play nicer with clang-format (indent everything with 2 tabs, instead of indent everything with 2 tabs *after* the open parenthesis.